apps/main/src/llamalend/features/user-position-history/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { t } from '@ui-kit/lib/i18n'
+import { Accordion } from '@ui-kit/shared/ui/Accordion'
+import { ParsedUserLendCollateralEvent } from './hooks/useUserLendCollateralEvents'
+import { UserEventsTable } from './UserEventsTable'
+
+type UserPositionHistoryProps = {
+  events: ParsedUserLendCollateralEvent[]
+  isLoading: boolean
+  isError: boolean
+}
+
+export const UserPositionHistory = ({ events, isLoading, isError }: UserPositionHistoryProps) => (
+  <Accordion title={t`Position History`} ghost defaultExpanded={false}>
```

Comment:
false is the default afaik?
```suggestion
  <Accordion title={t`Position History`} ghost>
```

Reply:
True, default is false